### ISSUE TITLE
How to show Thumbnail Local video?

### ISSUE THREAD

**Issue by Donhv** [lucasbento/react-native-thumbnail-video#8]
I want show Thumbnail for Local video
Example : requeri('./video/video.mp4')
Help me!

**Comment by lucasbento** [comment]
@Donhv: sorry, didn't see this issue before, there's currently no way of doing that, it supports only youtube video, would appreciate a PR though.

**Comment by LuongTruong** [comment]
@nicoara I agree, showing thumbnail and showing a paused video have a lot of difference. I am struggling with this because I cannot show thumbnail of MOV video in iOS

**Comment by nicoara** [comment]
I have found the code in the library react-native-thumbnail works.

**Comment by LuongTruong** [comment]
Hi @nicoara , I just realize that MOV movie thumbnail cannot be shown is not because of `Image` tag. It is because the return path of `react-native-image-picker` cannot be read by `Image` tag.

**Comment by lyxia** [comment]
Hi @truongluong1314520， Is there any solution now? In addition to pausing the video.I also use react-native-image-picker.

**Comment by lyxia** [comment]
Hi @nicoara, react-native-thumbnail resolve my problem, thanks.
